k8s/plugin/src/constant.rs

```diff
@@ -23,6 +12,11 @@ macro_rules! upgrade_labels {
     };
 }
 
+/// append the release name to k8s objects
+pub(crate) fn upgrade_group(release_name: String, name: &str) -> String {
```

Comment:
nit: why is release_name `String` and name `&str` ? Consider making both `&str` (would avoid the many clones down the line)